lib/autoload/filename_sanitizer.rb

```diff
@@ -0,0 +1,8 @@
+module FilenameSanitizer
```

Comment:
Yah I know, but since our server are unix based, should consider it valid ?

Sure, since there is a use case now !  But I think I still need two different methods ? one for path and one for filename(due to my last comment).